packages/suite/src/actions/wallet/send/sendFormThunks.ts

```diff
@@ -216,15 +215,15 @@ export const signAndPushSendFormTransactionThunk = createThunk(
         // this action is blocked by modalActions.preserve()
         dispatch(modalActions.preserve());
 
-        const { serializedTx } = await dispatch(
+        const response = await dispatch(
             signTransactionThunk({
-                formValues,
+                formState,
                 precomposedTransaction: enhancedPrecomposedTransaction,
                 selectedAccount,
             }),
-        ).unwrap();
+        );
 
-        if (!serializedTx) {
+        if (isRejected(response)) {
```

Comment:
yes, it is. It can be rejected for many reasons in the coin-specific thunks. The "root" `signTransactionThunk` passes this rejection here https://github.com/trezor/trezor-suite/blob/c1243dd17bbb7685a9d25e41b91ff5d3cbff3745/suite-common/wallet-core/src/send/sendFormThunks.ts#L469. The thunk is also possibly rejected at the top of the definition if the arguments are not correct. Did I answer your question?